subo/util/log.go

```diff
@@ -20,31 +21,36 @@ func (p *PrintLogger) LogInfo(msg string)  { LogInfo(msg) }
 func (p *PrintLogger) LogStart(msg string) { LogStart(msg) }
 func (p *PrintLogger) LogDone(msg string)  { LogDone(msg) }
 func (p *PrintLogger) LogFail(msg string)  { LogFail(msg) }
+func (p *PrintLogger) LogWarn(msg string)  { LogWarn(msg) }
 
-// LogInfo logs information
-func LogInfo(msg string) {
+// Keeping it DRY
+func Log(msg string) {
 	if _, exists := os.LookupEnv("SUBO_DOCKER"); !exists {
-		fmt.Println(fmt.Sprintf("ℹ️  %s", msg))
+		fmt.Println(msg)
 	}
 }
 
+// LogInfo logs information
+func LogInfo(msg string) {
+	Log(fmt.Sprintf("ℹ️  %s", msg))
+}
+
 // LogStart logs the start of something
 func LogStart(msg string) {
-	if _, exists := os.LookupEnv("SUBO_DOCKER"); !exists {
-		fmt.Println(fmt.Sprintf("⏩ START: %s", msg))
-	}
+	Log(fmt.Sprintf("⏩ START: %s", msg))
 }
 
 // LogDone logs the success of something
 func LogDone(msg string) {
-	if _, exists := os.LookupEnv("SUBO_DOCKER"); !exists {
-		fmt.Println(fmt.Sprintf("✅ DONE: %s", msg))
-	}
+	Log(fmt.Sprintf("✅ DONE: %s", msg))
 }
 
 // LogFail logs the failure of something
 func LogFail(msg string) {
-	if _, exists := os.LookupEnv("SUBO_DOCKER"); !exists {
-		fmt.Println(fmt.Sprintf("🚫 FAILED: %s", msg))
-	}
+	Log(fmt.Sprintf("🚫 FAILED: %s", msg))
+}
+
+// LogWarn logs a warning from something
+func LogWarn(msg string) {
+	Log(fmt.Sprintf("⚠️ FAILED: %s", msg))
```

Comment:
I tried inputting the warning emoji (⚠️) from both Chrome and VS Code, but it doesn't display it properly. Not sure why!

Reply:
It appears correctly for me :)